src/components/client/FormCreateUser.tsx

```diff
@@ -0,0 +1,97 @@
+import { useMutation } from "@tanstack/react-query";
+import { postUser } from "../../apis/users.api";
+import { useState } from "react";
+import {  UserPayload } from "../../types";
+
+function FormCreateUser() {
+  const [formData, setFormData] = useState({
```

Comment:
For the form data management, you can consider using [react hook form](https://react-hook-form.com/)